From private repo

v-anpasi opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

yujhongmicrosoft committed:
Delete documentation-government-function1.png

yujhongmicrosoft committed:
Delete documentation-government-function10.png

yujhongmicrosoft committed:
Delete documentation-government-function11.png

yujhongmicrosoft committed:
Delete documentation-government-function2.png

yujhongmicrosoft committed:
Delete documentation-government-function3.png

yujhongmicrosoft committed:
Delete documentation-government-function4.png

yujhongmicrosoft committed:
Delete documentation-government-function5.png

yujhongmicrosoft committed:
Delete documentation-government-function6.png

yujhongmicrosoft committed:
Delete documentation-government-function7.png

yujhongmicrosoft committed:
Delete documentation-government-function8.png

yujhongmicrosoft committed:
Delete documentation-government-function9.png

yujhongmicrosoft committed:
Delete documentation-government-functions-createvs.png

yujhongmicrosoft committed:
Update documentation-government-services.md

yujhongmicrosoft committed:
Update documentation-government-howto-deploy-webandmobile.md

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into custom-connectors

yujhongmicrosoft committed:
Merge branch 'master' into availableservices

ecfan committed:
Update art

ecfan committed:
Update art, add info about triggers

ecfan committed:
Fix typo

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into custom-connectors

ecfan committed:
Updates

yujhongmicrosoft committed:
Delete documentation-government-howto-deploy-webandmobile-openapp-new3.png

yujhongmicrosoft committed:
Delete documentation-government-howto-deploy-webandmobile-openapp-new5.png

yujhongmicrosoft committed:
Add files via upload

yujhongmicrosoft committed:
Update documentation-government-services.md

squillace committed:
cleaning up the PR

squillace committed:
updated code brackets

swkrish committed:
Merge pull request #11 from MicrosoftDocs/master

private branch

swkrish committed:
added a couple of limitations

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-sso.md

yujhongmicrosoft committed:
Merge branch 'master' into availableservices

squillace committed:
Update container-service-draft-up.md

HJToland3 committed:
connect article tweaks pt1

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

HJToland3 committed:
connect article tweaks Pt 1a

tfitzmac committed:
changed file names

Commit:
initial preview

tfitzmac committed:
fixed validation

Commit:
minor revision

HJToland3 committed:
Connect topic tweaks Pt2

Commit:
fixed author not found, tip placement

Commit:
class wording change

adegeo committed:
created 2 new samples

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-faq.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-sso-quick-start.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Update active-directory-aadconnect-user-signin.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

adegeo committed:
Fixed weird link problem

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-smart-lockout.md

swkrish committed:
Update active-directory-aadconnect-sso-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-upgrade-preview-authentication-agents.md

swkrish committed:
Update active-directory-aadconnect-sso-quick-start.md

genlin committed:
add questions for connectivity article

genlin committed:
add questions

genlin committed:
edit the articles

genlin committed:
fix the link issue

genlin committed:
correct typo

genlin committed:
fix the format issue

genlin committed:
correct typo

ecfan committed:
PM updates for validation phases

ecfan committed:
Formatting fixes

ecfan committed:
Merge branch 'custom-connectors' of https://github.com/ecfan/azure-docs-pr into custom-connectors

YuriDio committed:
updating articles

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

YuriDio committed:
updating articles

YuriDio committed:
fixing links

YuriDio committed:
fixing acrolinx

YuriDio committed:
fixing TOC

alkohli committed:
add a draft faq to move to Azure portal

ecfan committed:
Resolve conflicts

YuriDio committed:
fixing acrolinx 2

Commit:
ca79

YuriDio committed:
fixing article

divyaswarnkar committed:
batch1

Commit:
added topics and artwork for multi-vm update and sccm integration

divyaswarnkar committed:
batch2

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-how-it-works.md

ecfan committed:
Formatting fixes

ecfan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into custom-connectors

Commit:
added screenshots for update mgmt multi

Commit:
started multi-vm topic

ecfan committed:
Formatting fixes and added anchors

mahesh-unnikrishnan committed:
Update screenshots for the DNS article.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
ca79

tfitzmac committed:
updated portal images

YuriDio committed:
fixing article

mtillman committed:
Add PIM for Azure RBAC

alkohli committed:
finetune 1

Commit:
finished topics for multi-vm and sccm integration

mtillman committed:
minor edits

Commit:
fixed broken links

ecfan committed:
Add links

mtillman committed:
skwan edits

Commit:
fixed broken links

Commit:
fixed broken link

mmacy committed:
[aci] new: PowerShell quickstart

ecfan committed:
Update dates and add links

ecfan committed:
Formatting fixes, add links

ecfan committed:
Fix broken link

genlin committed:
remove locale

Commit:
made fixes per PR feedback

Commit:
fix content based on feedback

ecfan committed:
Formatting tweak

ecfan committed:
Formatting tweak

mumian committed:
update

mumian committed:
update

mumian committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into stale05-0825

Commit:
fix errors

Commit:
Merge branch 'release-ignite-automation' of https://github.com/Microsoft/azure-docs-pr into ignite-update-mgmt

Commit:
Merge branch 'release-ignite-automation' of https://github.com/microsoft/azure-docs-pr into ignite-update-mgmt

BryanLa committed:
updates

Commit:
made fixes to pass acrolynx

alkohli committed:
fine tune2

Commit:
fixed grammatical errors per acrolynx

mumian committed:
incorporate reviewer feedback

alkohli committed:
fine tune 3

alkohli committed:
Acrolinx changes

mumian committed:
update

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

HJToland3 committed:
url tweaks to remove locale references

mumian committed:
update link

ecfan committed:
Fix PR issues

ecfan committed:
Fix PR issues

mmacy committed:
[acs] update service principal scope requirements

saghorpa committed:
changes to remove log dir from snapshot 09212017

alkohli committed:
fine tune4

alkohli committed:
fine tune 5

alkohli committed:
fine tune 5

alkohli committed:
fine tune 7

Commit:
Java-Java update

Commit:
Add use-hsm-with-sdk

Commit:
Update toc.yml

mmacy committed:
[acs] update service principal scope requirements

mmacy committed:
[acs] update service principal scope requirements

Commit:
Change title

Juliako committed:
new files

mtillman committed:
add CA for Azure mgmt

Commit:
initial revision

Commit:
Removing images which were moved to a different repo

Commit:
fix errors

Juliako committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into APIMUpdates

Juliako committed:
fixed errors

mahesh-unnikrishnan committed:
Update AAD-DS references & terminology.

mahesh-unnikrishnan committed:
Update hdinsight-domain-joined-architecture.md

mahesh-unnikrishnan committed:
Update hdinsight-domain-joined-architecture.md

mahesh-unnikrishnan committed:
Update hdinsight-domain-joined-architecture.md

ecfan committed:
Update limits

Commit:
revision per @dominicbetts

Commit:
Azure AD and LinkedIn content FOR IGNITE

mtillman committed:
fix caution box; update art

HeidiSteen committed:
moved supported data sources to a separate article

Commit:
fied merge conflict in automation toc

Commit:
fixed merge conflict in toc

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta-api

HeidiSteen committed:
tweak

Commit:
fixed toc-removed conflice

mtillman committed:
skwan edits

mtillman committed:
fixed art

mtillman committed:
acrolinx edits

alkohli committed:
tech review

mtillman committed:
the right art now

Commit:
revision per @v-thepet

mtillman committed:
remove Azure from TOC entry

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

alkohli committed:
last pass for acrolinx

HJToland3 committed:
final tweaks, I hope!

cherylmc committed:
format

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta-api

HeidiSteen committed:
fixed a link

jluk committed:
fix error of draft to helm

MikeRayMSFT committed:
Attempt to fix broken link.

MikeRayMSFT committed:
Remove forward slash

ecfan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into custom-connectors

Commit:
Moved Tutorial #5 to How-tos

hning86 committed:
broken quick start link

ErikjeMS committed:
deleting new file, to be re-added later

This feature was cut at the last minute.

ErikjeMS committed:
delete article, to be re-added later

This feature was cut from Ignite at the last minute. Will re-add it in a week or two.

ErikjeMS committed:
delete article, to be re-added later

This Ignite feature was cut at the last minute. Will be re-added in a week or two

ErikjeMS committed:
3 articles deleted, to be re-added later

Ignite features cut at the last minute. Will be re-added later.

ErikjeMS committed:
redirects for deleted articles

Ignite features postponed at the last minute. Will be re-added in a week or two

czeumault committed:
Update index.md

czeumault committed:
Merge pull request #23945 from ErikjeMS/master

Features cut at last minute, deleting articles

czeumault committed:
Merge pull request #23943 from hning86/patch-2

fix broken quick start link

czeumault committed:
Fix broken link

czeumault committed:
Update index.md

Commit:
updating eula version

Commit:
Merge pull request #23947 from sayantancs/master

updating eula version

czeumault committed:
Merge pull request #23946 from MicrosoftDocs/czeumault-patch-27

Update index.md

GitHubber17 committed:
Merge pull request #23930 from SyntaxC4/patch-1

Moved Tutorial #5 to How-tos

GitHubber17 committed:
Merge pull request #20622 from ecfan/custom-connectors

New custom connectors content - Publish post-Ignite on 9/25 Monday afternoon, if poss

GitHubber17 committed:
Merge pull request #21344 from vigunase/password-change

Password change

GitHubber17 committed:
Merge pull request #22365 from yujhongmicrosoft/availableservices

Availableservices

ktoliver committed:
Merge pull request #23554 from eslesar/ignite-update-mgmt

Added topics for SCCM integration and multi-vm update management

GitHubber17 committed:
Merge pull request #22768 from squillace/draft-seven

updating the draft article to draft .7

GitHubber17 committed:
Merge pull request #23292 from tfitzmac/0919managedapps

changed file names

GitHubber17 committed:
Merge pull request #23375 from Thraka/sf-samples

created 2 new samples for service fabric

ktoliver committed:
Merge pull request #23389 from swkrish/master

Addressed latest customer feedback + clean up for GA announcement

ktoliver committed:
Merge pull request #23444 from YuriDio/AugustUpdates_24

August updates 24

rmca14 committed:
Merge pull request #23399 from genlin/master100

add questions for cloud service FAQ article

ktoliver committed:
Merge pull request #23478 from MarkusVi/ca79

ca79

ktoliver committed:
Merge pull request #23487 from divyaswarnkar/batch

batch1

rmca14 committed:
Merge pull request #23304 from msebolt/iot-hub-java-node-firmware-update

initial preview

GitHubber17 committed:
Merge pull request #23523 from mahesh-unnikrishnan/master

update screenshots for the DNS article.

jluk committed:
Merge branch 'master' into patch-1

ktoliver committed:
Merge pull request #23646 from mumian/stale05-0825

Stale05 0825

ktoliver committed:
Merge pull request #23658 from BryanLa/bryanla-msi-toc

MSI updates

GitHubber17 committed:
Merge pull request #23660 from alkohli/SSFAQ

FAQ: Move to Azure portal - Go live by 9/25, Monday 3 pm (or before)

GitHubber17 committed:
Merge pull request #23742 from yzhong94/master

Add how to use HSM with SDK doc

GitHubber17 committed:
Merge pull request #23756 from Juliako/APIMUpdates

new files

GitHubber17 committed:
Merge pull request #23758 from mtillman/mtillman921mstr

Add PIM and CA articles for Azure resources

rmca14 committed:
Merge pull request #23721 from mmacy/acs-sp-scope-fix

[acs] update service principal scope requirements

GitHubber17 committed:
Merge pull request #23759 from rmcmurray/20170908b

Removing images which were moved to a different repo

GitHubber17 committed:
Merge pull request #23783 from HeidiSteen/heidist-ta-api

moved supported data sources to a separate article

ktoliver committed:
Merge pull request #23722 from saghorpa/HADRChangesforlogsnapremoval09212017

changes to remove log dir from snapshot 09212017

rmca14 committed:
Merge pull request #23761 from mahesh-unnikrishnan/patch-16

Update AAD-DS references & terminology.

GitHubber17 committed:
Merge pull request #23788 from HJToland3/localworking

Localworking

GitHubber17 committed:
Merge pull request #23815 from cherylmc/fixtable

format

Commit:
fix important broken link

Commit:
Merge pull request #23818 from jluk/patch-1

fix error of draft to helm

mmacy committed:
[aci][toc] resolve TOC merge conflict

PRMerger20 committed:
Merge pull request #23951 from cbrooksmsft/patch-2

fix important broken link

GitHubber17 committed:
Merge pull request #23573 from mmacy/aci-quickstart-powershell

[aci] new: PowerShell quickstart

Commit:
removed incorrect redirect

Commit:
removed redirect

Commit:
corrected redirects

Commit:
Azure Stack Operator metadata

Lisaco88 committed:
Merge pull request #23777 from jeffgilb/jeffIgnite2

Azure AD and LinkedIn content FOR IGNITE

garyericson committed:
Redirecting broken link on ACOM page

AnithaAdusumilli committed:
Fixes for links, formatting headers

tynevi committed:
Merge pull request #23954 from georgewallace/master

removed incorrect redirect

PRMerger12 committed:
Merge pull request #23955 from kriscrider/metadata_azurestack

Azure Stack Operator metadata

tynevi committed:
Merge pull request #23957 from garyericson/patch-7

Redirecting broken link on ACOM page

Lisaco88 committed:
Merge pull request #23821 from MikeRayMSFT/20170922_FixTocForSQLVM

Attempt to fix broken link.

PRMerger18 committed:
Merge pull request #23959 from AnithaAdusumilli/patch-1

Fixes for links, formatting headers

v-anpasi committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into FromPrivateRepo
